js/common/grids.d.ts

```diff
@@ -519,7 +519,7 @@ export type ColumnChooserMode = 'dragAndDrop' | 'select';
  * @namespace DevExpress.common.grids
  */
 export type ColumnCustomizeTextArg = {
-  value?: string | number | Date;
+  value?: any;
```

Comment:
Better add generic parameter instead:
```suggestion
  value?: TValue;
```
Is it correct to pass in `ColumnBase`'s `TRowData` ?

Reply:
Discussed in private, we gonna try fix all such cases in the future